src/data_gradients/feature_extractors/object_detection/bounding_boxes_area.py

```diff
@@ -84,7 +87,9 @@ def aggregate(self) -> Feature:
         json = {}
         for split in df["split"].unique():
             basic_stats = dict(df[df["split"] == split]["relative_bbox_area"].describe())
-            json[split] = {**basic_stats, "histogram_per_class": dict_bincount[split]}
+            json[split] = {**basic_stats,
+                           "histogram_per_class_area": dict_bincount_area[split],
+                           "histogram_per_class_area_perimeter": dict_bincount_area_perimeter[split]}
```

Comment:
I can see you did not apply black. Please run `black .` or `black myfile.py` to make sure it formats the same way we do for the rest of the repo

Reply:
`histogram_per_class_area_perimeter` -> `histogram_per_class_perimeter` No ? The area should be removed I guess

Reply:
I add black. thanks.
it's area devided by perimeter (no sqrt) - Do you have better naming?